### DISCUSSION TITLE
Could you add a new function (button) - new day?

### DISCUSSION THREAD

**Discussion by adrakadabra** [YetAnotherPomodoroApp/YAPA-2#94] [Ideas]
In a situation when you work all day and without turning off the computer you start working again the next day, the yapa counter (Competed today) is not reset, but continues to increase.

**Comment by floatas** [comment]
I never thought about this scenario, this is definitely a bug.
  
Note: When Pomodoro is completed update DB, and fetch a new count from DB.
